tests/src/JIT/Regression/JitBlue/GitHub_1323/GitHub_1323.cs

```diff
@@ -0,0 +1,22 @@
+using System;
```

Comment:
Can you add the MIT license header here?

// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
//